leptos_hotkeys/src/hotkeys_provider.rs

```diff
@@ -43,15 +32,16 @@ where
     T: ElementDescriptor + 'static + Clone,
 {
     #[cfg(not(feature = "ssr"))]
-    let active_ref_target: RwSignal<Option<EventTarget>> = RwSignal::new(None);
+    let active_ref_target: RwSignal<Option<web_sys::EventTarget>> = RwSignal::new(None);
 
     #[cfg(not(feature = "ssr"))]
-    let set_ref_target = Callback::new(move |target: Option<EventTarget>| {
+    let set_ref_target = Callback::new(move |target: Option<web_sys::EventTarget>| {
         active_ref_target.set(target);
     });
 
     #[cfg(not(feature = "ssr"))]
-    let pressed_keys: RwSignal<HashMap<String, KeyboardEvent>> = RwSignal::new(HashMap::new());
+    let pressed_keys: RwSignal<HashMap<String, web_sys::KeyboardEvent>> =
+        RwSignal::new(HashMap::new());
```

Comment:
The main problem is this. When `ssr`, this code should also run.

Reply:
thanks!!